lighthouse-core/scripts/i18n/collect-strings.js

```diff
@@ -634,13 +627,12 @@ if (require.main === module) {
 
   for (const folderWithStrings of foldersWithStrings) {
     console.log(`\n====\nCollecting strings from ${folderWithStrings}\n====`);
-    const moreStrings = collectAllStringsInDir(folderWithStrings);
-    Object.assign(strings, moreStrings);
+    collectAllStringsInDir(folderWithStrings, strings);
   }
 
   if (collisions > 0) {
     console.log(`MEANING COLLISION: ${collisions} string(s) have the same content.`);
-    assert.equal(collisions, 30, `The number of duplicate strings have changed, update this assertion if that is expected, or reword strings. Collisions: ${collisionStrings.join('\n')}`);
+    assert.equal(collisions, 32, `The number of duplicate strings have changed, update this assertion if that is expected, or reword strings. Collisions: ${collisionStrings.join('\n')}`);
```

Comment:
this is +2 for the collision between `"lighthouse-treemap/app/src/util.js | tableColumnName"` and `"lighthouse-core/lib/i18n/i18n.js | columnName"` we didn't add in #12441